salt/_grains/caasp_nodename.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+
+import salt.grains.core
+
+def hostname():
```

Comment:
Considering that the only function in this file is `hostname()`, wouldn't make more sense to name this file `caasp_hostname`?

Reply:
Actually, Does salt not require that the function name and file name matches?